ftrr/src/main/java/framgia/co/edu/ftrr/controller/div/RequestController.java

```diff
@@ -141,4 +142,14 @@ public ResponseEntity importRequestTrainees(@RequestParam("multipartFile") Multi
         }
     }
 
+    @PostMapping("/final-result")
+    public ResponseEntity<RequestDTO> updateFinalResults(@RequestBody RequestDTO requestDTO){
+        RequestDTO requestDTOResult = getRequestService().updateFinalResult(requestDTO);
```

Comment:
để an toàn hơn và nhanh trả về kết quả hơn, em check việc có tồn tại request ko, nếu nil thì báo lỗi luôn, nếu khác nill thì mới gọi line này chẳng hạn